### ISSUE TITLE
A question on weight decay.

### ISSUE THREAD

**Issue by kmbmjn** [shariqfarooq123/AdaBins#42]
Hi,
Thank you for sharing the code.
I would like to ask a question on the weight decay coefficient.

In your paper, you said, "For training, we use the AdamW optimizer [30] with weight-decay 10−2."
However, in your code(args_train_kitti_eigen.txt or args_train_nyu.txt), you wrote, "--wd 0.1"
Can you tell me which is correct?

Thanks.

**Comment by shariqfarooq123** [comment]
The final model provided in this repo is with wd of 0.1
